### PR TITLE
March newsletter

### PR DESCRIPTION
**Pull request by bethaitman**

### CODE REVIEW

File: docs/blog/newsletter-march-2022.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+​
+Still, it's more important for technical writers to know how to prepare documentation than every particular of their subject. Rather than knowing how to make API requests in several programming languages, for example, technical writers should know how to use Postman (or a similar tool) to configure example API requests in any supported language. From this perspective, "a little more than just enough" is the optimal balance of domain expertise for technical writers.
+
+---------------------------------
+Automation dreams - and realities
+---------------------------------
+
+A question asking which things you'd most like to automate about docs produced a plethora of wish lists and practical implementations. Here's the list of tasks our community is ready to hand off to our robot overlords, small or mighty:
```
Comment (hillaryfraley) [inline docs/blog/newsletter-march-2022.rst:29]: I love this intro!

File: docs/blog/newsletter-march-2022.rst [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+- Screenshots and changes to UI text.
+- Comprehensive and accurate lists of which features are available in which version of the product.
+- Release notes! This can be quite feasible, if you generate them from issues or pull requests.
+- Terminology checks ("linter type of things" as one contributor expansively put it), broken link checkers, and more text and formatting checks.
+- Copying tables from one format into another (notably to Markdown or HTML).
+- Docs builds and deployments (not everyone works with CI/CD!).
+- Slackbots to create and even help manage docs issues and reviews directly from Slack conversations. A couple of folks use these, and one person offered `an example from the UK's Government Digital Services <https://technology.blog.gov.uk/2020/09/25/keeping-tech-docs-up-to-date-with-daniel-the-manual-spaniel/>`_
+
+Lots of ideas here - the next step is to go forth and automate
```
Comment (hillaryfraley) [inline docs/blog/newsletter-march-2022.rst]:
```suggestion
Lots of ideas here - the next step is to go forth and automate!
```

**Review by hillaryfraley — APPROVED** [review]
Just one suggestion (although I'm not sure what type of punctuation works best there -- maybe an exclamation point, maybe not)

**Comment by bethaitman** [comment]
Thanks!

**Pull request merged**